plexapi/library.py

```diff
@@ -2849,6 +2861,137 @@ def _loadData(self, data):
         self.type = data.attrib.get('type')
 
 
+class HubManage(PlexObject):
```

Comment:
I think renaming to `ManagedHub` sounds better.
```suggestion
class ManagedHub(PlexObject):
```